TheMovieDbTests/TheMovieDbTests.swift

```diff
@@ -9,16 +9,27 @@ import XCTest
 @testable import TheMovieDb
 
 class TheMovieDbTests: XCTestCase {
+    var sut: MovieClient!
 
     override func setUpWithError() throws {
         // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        try super.setUpWithError()
+        sut = MovieClient()
```

Comment:
If MovieClient is not optional/throwable initialization would be clearer to override the setUp method